src/gt4sd/domains/materials/__init__.py

```diff
@@ -33,6 +34,8 @@
 # TODO: setting to str directly requires no wrapping, so wrong strings could be passed
 Protein = str  # NewType('Protein', str)
 SMILES = str  # NewType('SMILES', str)
+SELFIES = str  # NewType('SELFIES', str)
+COPOLYMER = str  # NewType('COPOLYMER', str)
```

Comment:
Since it's not an abbreviation:
```suggestion
Copolymer = str  # NewType('Copolymer', str)
```